src/vs/workbench/workbench.main.ts

```diff
@@ -525,3 +525,5 @@ import 'sql/workbench/parts/dashboard/browser/containers/dashboardContainer.cont
 import 'sql/workbench/parts/dashboard/browser/containers/dashboardNavSection.contribution';
 import 'sql/workbench/parts/dashboard/browser/containers/dashboardModelViewContainer.contribution';
 import 'sql/workbench/parts/dashboard/browser/core/dashboardTab.contribution';
+
+import 'sql/workbench/parts/commandLine/electron-main/commandLine.contribution';
```

Comment:
>electron-main [](start = 40, length = 13)

So side note - but do we have a list anywhere of what things belong under electron-main/browser/electron-browser/node/etc? If not that would be awesome to have...I know I at least don't recall off hand all the different ones and what belongs under each folder type.

Reply:
https://github.com/microsoft/vscode/wiki/Code-Organization